.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @zephyr-ai/security
```

Comment:
I'm adding the security team as a CODEOWNER since this repo gives us the ability to modify our public facing GitHub organizational page.